client/opentracing/src/main/java/org/hawkular/apm/client/opentracing/EnvironmentAwareTraceRecorder.java

```diff
@@ -38,10 +38,12 @@ public void record(Trace trace) {
         if (trace.getNodes() != null && trace.getNodes().size() > 0) {
             Node rootNode = trace.getNodes().get(0);
             if (this.deploymentMetaData != null) {
-                if (!rootNode.hasProperty(Constants.PROP_SERVICE_NAME)) {
+                if (!rootNode.hasProperty(Constants.PROP_SERVICE_NAME)
```

Comment:
The service name may be provided by the application (i.e. setting tag directly).